internal/impl/pop3/input_pop3.go

```diff
@@ -0,0 +1,269 @@
+package pop3
+
+import (
+	"context"
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/benthosdev/benthos/v4/public/service"
+	"github.com/knadh/go-pop3"
```

Comment:
We could, certainly, but then we'd need to implement the methods we're using, we'd basically need to copy-paste them. I think we could keep using it, at least for now. 
I'm also drafting another PR for the IMAP protocol and on that one, I'm using `https://github.com/emersion/go-imap`, maybe then I'll find out if I can use `github.com/emersion/go-message` directly, and add TLS certs support ... if needed.

Reply:
Sure, sounds good. Just thought this lib is still a bit young :)

Reply:
True, and not very active :D We can "re-implement", also known as shamelessly copying them, the methods if you think is safer, from what I could test they're working fine.

Reply:
Whichever you think is more maintainable. I don't really know... There's a new version of `github.com/emersion/go-message` from May which wasn't propagated to `github.com/knadh/go-pop3` yet, so I guess it will lag behind.